addons/sale/views/sale_portal_templates.xml

```diff
@@ -331,7 +344,7 @@
                         </t>
                         <div t-elif="sale_order.has_to_be_paid(True)" class="col-sm-auto mt8">
                             <a role="button" class="btn btn-primary" data-toggle="modal" data-target="#modalaccept" href="#">
-                                <i class="fa fa-check"/> <t t-if="not sale_order.signature">Accept &amp; Pay</t><t t-else="">Pay now</t>
+                                <i class="fa fa-check"/> <t t-if="not sale_order.signature or sale_order.require_payment">Accept &amp; Pay</t><t t-else="">Pay now</t>
```

Comment:
What is the purpose of this ? As we are already in an "has_to_be_paid" condition this should normally not be necessary. Am I wrong ?

Reply:
It was to say that the payment was also sometimes used as only way of confirmation and that "Accept" should also be visible in this case, but the condition is wrong for that in its current state.